packages/flutter/test/foundation/platform_utils.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+/// Replaces Windows-style path separators with Unix-style separators.
+String sanitizePaths(String path) {
```

Comment:
This name left me wondering what it was doing (and thinking that it was weird to map windows-style to unix-style when you're on Windows).  Maybe call this something like `toPosixPath`?